spec/support/have_content_tag_for_matcher.rb

```diff
@@ -1,9 +1,24 @@
 require 'rspec/expectations'
 require 'action_view/record_identifier'
 
+module ContentTag
+  module TestExampleHelpers
```

Comment:
It seems `ContentTag` and `TestExampleHelpers` need to be defined previously for that to work. I'm just using these for namespacing.

I'm exposing `content_tag_selector` to features as well, so I thought it would make sense for it to be standalone.